dev_cmds/setup-dev.ts

```diff
@@ -44,8 +38,7 @@ async function main() {
       },
     });
   } catch {
-    process.stderr.write("Could not setup the dev environment!");
-    process.exit(1);
+    throw new Error("Could not setup the dev environment!");
```

Comment:
hmmm, we should probably include some details about what the problem actually was here -- e.g. catch and include the inner Error (or message)